packages/vite/src/node/plugins/esbuild.ts

```diff
@@ -147,7 +147,7 @@ export async function transformWithEsbuild(
       ...result,
       map
     }
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
Isn't this a TS 4.4 feature? We will try to set `useUnknownInCatchVariables` to `false`

https://github.com/vitejs/vite/pull/4932/files#diff-268c58d28954957768302d8eff57fc09815d313ad5ea421f34e1736d6d74f8e4R10

For now I would suggest to use the TS version currently used by the project until `@microsoft/api-extractor` is updated.

Reply:
@Shinigami92 I see this is used in other parts of the code. I would like to merge this one so we can do a beta.1 and start having some feedback. Let's resolve this one and others in another PR if that is ok

Reply:
Ah okay, if it is used already, then nevermind 👍